src/solver/optimisation/opt_appel_solveur_quadratique.cpp

```diff
@@ -183,3 +183,153 @@ bool OPT_AppelDuSolveurQuadratique(PROBLEME_ANTARES_A_RESOUDRE* ProblemeAResoudr
         return false;
     }
 }
+
+void setInteriorPointProblem(PROBLEME_ANTARES_A_RESOUDRE* ProblemeAResoudre,
+                             PROBLEME_POINT_INTERIEUR& Probleme)
+{
+    double ToleranceSurLAdmissibilite;
+    int ChoixToleranceParDefautSurLAdmissibilite;
+    double ToleranceSurLaStationnarite;
+    int ChoixToleranceParDefautSurLaStationnarite;
```

Comment:
Move all adq-patch stuff to **adequacy_patch_curtailment_sharing_problem.cpp**.

Create a directory called adequacy_patch_csr/